fastly/alerts_test.go

```diff
@@ -23,14 +23,26 @@ func TestClient_FastlyAlerts(t *testing.T) {
 		Dimensions:         testDimensions,
 		EvaluationStrategy: testEvaluationStrategy,
 		IntegrationIDs:     []string{},
-		Metric:             ToPointer("status_5xx"),
+		Metric:             ToPointer("status_5xx_rate"),
 		Name:               ToPointer("test name"),
 		ServiceID:          ToPointer(TestDeliveryServiceID),
 		Source:             ToPointer("domains"),
 	}
 
-	// Test
 	var err error
+
+	// Enable Product - Domain Inspector
+	Record(t, "alerts/enable_required_product", func(c *Client) {
```

Comment:
We either enable Domain Inspector or we remove `TestClient_FastlyAlerts` entirely.